src/projects.rs

```diff
@@ -263,8 +263,12 @@ pub fn add_item_to_project(
     config: &Config,
     content: String,
     project: &str,
+    priority: String,
```

Comment:
I think that it would be better if we pass the enum around rather than the string, thoughts?

Reply:
yeah, missed it, thanks